docs/api/ReactWrapper/invoke.md

```diff
@@ -1,6 +1,8 @@
 # `.invoke(propName)(...args) => Any`
 
 Invokes a function prop.
+Note that `.invoke` will wrap your handler with [`ReactTestUtils.act`](https://reactjs.org/docs/test-utils.html#act) and call `.update()` automatically. It would
```

Comment:
```suggestion
Note that in React 16.8+, `.invoke` will wrap your handler with [`ReactTestUtils.act`](https://reactjs.org/docs/test-utils.html#act) and call `.update()` automatically. This may be more ergonomic than directly invoking a `.props()` function, since you won't have to wrap it in `act` yourself.
```